frontend/src/static/js/main.js

```diff
@@ -164,6 +203,26 @@ window.app = new window.Vue({
     generateKey(dataObj) {
       return JSON.stringify(dataObj);
     },
+
+    hasCommits(info) {
+      return window.REPOS[info.repo].commits.authorFinalContributionMap[info.author] > 0;
+    },
+
+    removeInfoHash(hashObj) {
+      const tabKeys = ['tabAuthor', 'tabLocation'];
+      return hashObj.filter((value) => {
+        const key = value.split('=')[0];
+        return !tabKeys.includes(key);
+      });
+    },
```

Comment:
create a equivalent `window.rmHash` that does the opposite as `window.addHash`.

Reply:
I didn't implement `removeHash`, because the hash cannot be updated consecutively like
```
addHash('tabOpen', this.isTabActive);
removeHash('tabAuthor');
removeHash('tabLocation');
```
(efficiency probs I think), the hash will not change. 
So I generate the params first, and update the hash directly.

Reply:
In that case do it such that there is addHash, rmHash and updateHash.

Alter addHash and rmHash to only handle hashParam.

Then updateHash to update the actual hash.

Cos when I am adding a new view, I shouldnt have to create another function inside the main vue app to handle the hash update for that tab.

If you need i could clean up the hash implementation first (my original approach aint the best) then you pull and cont from there?